examples/bri-3/src/bri/merkleTree/agents/merkleTreeStorage.agent.ts

```diff
@@ -55,6 +55,9 @@ export class MerkleTreeStorageAgent extends PrismaService {
       where: { id: merkleTree.id },
     });
 
+    // TODO: Mil5 - Mapping from a prisma client type to a domain object is not working.
```

Comment:
open issue or will be resolved before PR merged?

Reply:
@Therecanbeonlyone1969  This code was copied from Ognjen's PR. This question has been addressed in PR #732